### PR TITLE
Create fetch-production-db

### PR DESCRIPTION
**Pull request by kaystrobach**
example for fetch the production db from the remote server with ddev

### CODE REVIEW

**Comment by rfay** [comment]
Thanks!

File: hook-examples/fetchproductiondb/fetchproductiondb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+## Description: Open PHPStorm with the current project
+## Usage: fetchproductiondb
+## Example: "ddev fetchprodctiondb"
+
+# Example is macOS-specific, but easy to adapt to any OS
+ssh -t user@host 'mysqldump --defaults-file=~/.mysql/project.cnf databasename' > productionsqldump.sql
+ddev import-db --src=productionsqldump.sql
```
Comment (rfay) [inline hook-examples/fetchproductiondb/fetchproductiondb]: FYI `ddev import-db` (and also `ddev mysql`) are happy to accept from stdin, so it's not actually necessary to dump to a file first. (Or end up having to delete the file)

Comment (kaystrobach) [reply hook-examples/fetchproductiondb/fetchproductiondb]: mhm ok, we decided to have the step as we can then easier debug the output by simply adding a pause or commenting the rm - but i will think about that

Comment (kaystrobach) [reply hook-examples/fetchproductiondb/fetchproductiondb]: the web command sounds like a good plan - will check it out

**Comment by rfay** [comment]
Another suggestion might be to use "pullprod" or something as the verb here, matching `ddev pull`

**Comment by rfay** [comment]
Rebased, Moved to custom command, made several edits, moved config into variables. Hope you like it! Thanks for this!

**Pull request merged**